views/cardView.json

```diff
@@ -0,0 +1,173 @@
+{
+    "view": {
+        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+        "type": "AdaptiveCard",
+        "version": "1.0",
```

Comment:
please update to : schemaVersion": 1,

for reference please follow : https://github.com/OfficeDev/microsoft-teams-apps-survey/blob/main/views/cardView.json